app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] after adding the foreign key in the user class, you should change this belongs_to part to look similar to the post class.